README.rst

```diff
@@ -170,9 +177,9 @@ Roll your own context_parser
     logger = pypyr.log.logger.get_logger(__name__)
 
 
-    def get_parsed_context(context):
+    def get_parsed_context(context_string):
         """This is the signature for a context parser. Input context is the string received from pypyr --context 'value here'"""
-        assert context, ("pipeline must be invoked with --context set.")
+        assert context_string, ("pipeline must be invoked with --context set.")
```

Comment:
If the string is actually serialized context then that's OK, but the command line context argument could be more clearly labelled, e.g. as context_arg.

Correct me if I have got this totally wrong.